tests/components/steam_online/__init__.py

```diff
@@ -62,16 +64,32 @@ def ISteamUser(self) -> None:
 
     def GetFriendList(self, steamid: str) -> dict:
         """Get friend list."""
-        return {"friendslist": {"friends": [{"steamid": ACCOUNT_2}]}}
-
-    def GetPlayerSummaries(self, steamids: str) -> dict:
+        fake_friends = []
+        for _i in range(0, 400):
+            fake_friends.append(
+                {"steamid": "".join(random.choices(string.digits, k=len(ACCOUNT_1)))}
+            )
+        return {"friendslist": {"friends": [{"steamid": ACCOUNT_2}, *fake_friends]}}
```

Comment:
I think this would be a bit more easy to grok if instead the first account (`ACCOUNT_2`) was in the `fake_friends` list when initialized and then you appended to that.